src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v1-monitoring-integration.mdx

```diff
@@ -12,6 +12,10 @@ redirects:
 freshnessValidatedDate: never
 ---
 
+<Callout variant="tip">
+  API Gateway V1 exclusively facilitates API polling, while metrics for [API Gateway V2](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v2-monitoring-integration) are streamed through CloudWatch Metric Streams.
```

Comment:
Change to "API Gateway v1 metrics are available exclusively through the API polling integration for Amazon API Gateway. [API Gateway v2 metrics](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v2-monitoring-integration) are available exclusively through the CloudWatch Metric Streams integration with AWS."

Reply:
@glennthomas1, i made the suggested changes.